src/main/java/com/spotify/github/v3/clients/OrganisationClient.java

```diff
@@ -53,62 +46,11 @@ static OrganisationClient create(final GitHubClient github, final String org) {
   }
 
   /**
-   * Create a team in an organisation.
-   *
-   * @param request create team request
-   * @return team
-   */
-  public CompletableFuture<Team> createTeam(final TeamCreate request) {
-    final String path = String.format(TEAM_TEMPLATE, org);
-    log.debug("Creating team in: " + path);
-    return github.post(path, github.json().toJsonUnchecked(request), Team.class);
-  }
-
-  /**
-   * Get a specific team in an organisation.
-   *
-   * @param slug slug of the team name
-   * @return team
-   */
-  public CompletableFuture<Team> getTeam(final String slug) {
-    final String path = String.format(TEAM_SLUG_TEMPLATE, org, slug);
-    log.debug("Fetching team from " + path);
-    return github.request(path, Team.class);
-  }
-
-  /**
-   * List teams within an organisation.
-   *
-   * @return list of all teams in an organisation
-   */
-  public CompletableFuture<List<Team>> listTeams() {
-    final String path = String.format(TEAM_TEMPLATE, org);
-    log.debug("Fetching teams from " + path);
-    return github.request(path, LIST_TEAMS);
-  }
-
-  /**
-   * Update a team in an organisation.
-   *
-   * @param request update team request
-   * @param slug slug of the team name
-   * @return team
-   */
-  public CompletableFuture<Team> updateTeam(final TeamCreate request, final String slug) {
-    final String path = String.format(TEAM_SLUG_TEMPLATE, org, slug);
-    log.debug("Updating team in: " + path);
-    return github.patch(path, github.json().toJsonUnchecked(request), Team.class);
-  }
-
-  /**
-   * Delete a specific team in an organisation.
+   * Create a Teams API client.
    *
-   * @param slug slug of the team name
-   * @return team
+   * @return Teams API client
    */
-  public CompletableFuture<Void> deleteTeam(final String slug) {
-    final String path = String.format(TEAM_SLUG_TEMPLATE, org, slug);
-    log.debug("Deleting team from: " + path);
-    return github.delete(path).thenAccept(IGNORE_RESPONSE_CONSUMER);
+  public TeamClient createTeamClient(final GitHubClient github, final String org) {
+    return TeamClient.create(github, org);
```

Comment:
This looks like a breaking change from the earlier version 🤔

Reply:
I had the same thought, but I think it's fine since we just released the code that we're breaking here last week (https://github.com/spotify/github-java-client/pull/135), therefore nobody is using it yet probably.

Reply:
Good point that hopefully no one is using it. Can we also unpublish the earlier version somehow? In case someone decide to use it in future?
Other than that I agree it's not worth the effort to fix the breakage.